clients/src/main/resources/common/message/BrokerRegistrationRequest.json

```diff
@@ -14,14 +14,16 @@
 // limitations under the License.
 
 // Version 1 adds Zk broker epoch to the request if the broker is migrating from Zk mode to KRaft mode.
-//
+
 // Version 2 adds the PreviousBrokerEpoch for the KIP-966
+
+// Version 3 adds LogDirs for KIP-858
 {
   "apiKey":62,
   "type": "request",
   "listeners": ["controller"],
   "name": "BrokerRegistrationRequest",
-  "validVersions": "0-2",
+  "validVersions": "0-3",
```

Comment:
What's the reasoning behind doing two bumps of this API version within the same Kafka release? 3.6 is currently on `"validVersions": "0-3"`, so it looks like Kafka 3.7.0 will introduce both versions 2 and 3 of this API.

Reply:
@soarez Ah, thanks for pointing this out 🙏